packages/backend/src/config/app.ts

```diff
@@ -75,6 +76,7 @@ const appConfig: AppConfig = {
   webhookUrl,
   requestBodySizeLimit: '1mb',
   postmanApiKey: process.env.POSTMAN_API_KEY,
+  formsgApiKey: process.env.FORMSG_API_KEY,
```

Comment:
out of scope: actually why do we have API keys here when they are only used in their own specific app <.<.  Should we just make the app code access env directly

(Also it doesn't seem like postman key is used at all, tbh)

Reply:
Adding it here for env var checking. I.e. the app should not be able to start up successfully, and rollback if the env var is missing. 

Postman key used for sending OTPs.

Reply:
oh ya I meant our send email helper directly accesses `process.env.POSTMAN_API_KEY` instead of using `app.postmanApiKey`, hence the initial comment

Anyway NBD, this works too.

Reply:
oh shucks lemme fix that